clang/lib/CodeGen/CGSYCLRuntime.cpp

```diff
@@ -78,9 +78,9 @@ bool CGSYCLRuntime::actOnFunctionStart(const FunctionDecl &FD,
       // be handled specially in the SYCL lowering pass
       F.setMetadata(PFWI_MD_ID, llvm::MDNode::get(F.getContext(), {}));
     break;
```

Comment:
```suggestion
    break;
  default:
    // fall through
```

Reply:
Adding the default here will lead to the same build error.

Reply:
Please see what I am proposing as a change.

Reply:
What is the build error? Would be good to add this info to the PR description as well.

Reply:
This is a known issue about the default case of switch statement in clang. 
Please see:
https://godbolt.org/z/5jvKjz
and
https://godbolt.org/z/sY6bdE
I can propose this solution.
https://godbolt.org/z/oqY8aG

Reply:
One more thing to confirm this change:
https://llvm.org/docs/CodingStandards.html#don-t-use-default-labels-in-fully-covered-switches-over-enumerations

Reply:
The combination of these 2 flags that will have the effect of a static. So there is really no need to add the assert as I was proposing in https://godbolt.org/z/oqY8aG.
See https://godbolt.org/z/PqE7Mf

Reply:
I see, thanks

Reply:
> So there is really no need to add the assert

I agree with this.